.github/workflows/integration-tests.yml

```
@@ -0,0 +1,331 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests", "642-integration-tests-add-tests-for-aliens", "648-integrationtests-add-consistency-test-for-random-putgetdelete-requests-with-bobt-tool" ] #REMOVE WHEN ON MASTER
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+(.alpha.[0-9]+)*"
+  pull_request_review:
+    types: [submitted]
+  workflow_dispatch:
+    inputs:
+      bob_nodes_amount:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_vdisk_count:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas_amount:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload_size:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_record_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '10000'
+      bob_first_index:
+        description: 'Index of the first record to proceed during tests'
+        required: true
+        default: '0'
+      bob_bobt_count:
+        description: 'Amount of iterations for chain operation test'
+        required: true
+        default: '10000'
```

Count for `bobt` tests should be `~ 5 * (last_index - first_index)`.
```suggestion
        default: '100000'
```